File: Programmers/문자열 다루기 기본.js
// https://programmers.co.kr/learn/courses/30/lessons/12918

function solution(s) {
  return (s.length === 4 || s.length === 6) && !s.split("").filter((v) => Number.isNaN(parseInt(v))).length;
}

// const s = "a234";
// const s = "1234";
const s = "02345a";

console.log(solution(s));

File: Programmers/문자열 다루기 기본.swift
//https://programmers.co.kr/learn/courses/30/lessons/12918?language=swift
//문자열 다루기 기본

func solution(_ s:String) -> Bool {
    guard s.count == 4 || s.count == 6 else {
        return false
    }
    if Int(s) != nil {
        return true
    } else { return false }
}

//let s: String = "a234"
////false

let s: String = "1234"
//true

print(solution(s))
